test/unittests/t_ingestion_stress.cc

```diff
@@ -467,18 +469,33 @@ TEST_F(T_IngestionStress, ProcessMultipleFilesInSeparateWavesSlow) {
   pipeline.Spawn();
 
   // first wave...
-  pipeline.Process(GetEmptyFile(), true);
+  std::string empty_path_1(GetEmptyFile());
+  IngestionSource *source_empty_1 = new FileIngestionSource(empty_path_1);
+
+  pipeline.Process(source_empty_1, true);
   pipeline.WaitFor();
   CheckHash(uploader_->results, GetEmptyFileBulkHash());
   uploader_->ClearResults();
 
   // second wave...
   // some small and medium sized files with file chunking enabled
   // one big file without file chunking
-  pipeline.Process(GetEmptyFile(), true);
-  pipeline.Process(GetSmallFile(), true);
-  pipeline.Process(GetBigFile(), true);
-  pipeline.Process(GetHugeFile(), false, shash::kSuffixCatalog);
+  std::string empty_path_2(GetEmptyFile());
```

Comment:
This could be more concise as `pipeline.Process(new FileIngestionSource(GetEmptyFile()), ...)` to avoid the extra definition of a `string` and a `IngestionSource` variable.